common-test/src/main/java/io/strimzi/test/ClusterController.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.test;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Annotation for test classes or methods run via {@code @RunWith(StrimziRunner.class)}
+ * which causes that runner to create a cluster controller before, and a cluster controller after,
```

Comment:
Looks like you forgot to say "delete" in this sentence. "and delete cluster controller after"